src/components/Workspace.jsx

```diff
@@ -330,8 +366,41 @@ class Workspace extends React.Component {
     );
   }
 
+  _storeColumnRef(index, column) {
+    this.columnRefs[index] = column;
+  }
+
+  _storeDividerRef(divider) {
+    this.dividerRef = divider;
+  }
+
+  _handleDividerStart() {
+    this.props.dispatch(startDragColumnDivider());
+  }
+
+  _handleDividerStop() {
+    this.props.dispatch(stopDragColumnDivider());
+  }
+
+  _handleDividerDrag(_, {deltaX, lastX, x}) {
+    this.props.dispatch(dragColumnDivider({
+      columnWidths: getNodeWidths(this.columnRefs),
+      dividerWidth: getNodeWidth(this.dividerRef),
```

Comment:
I could see an argument for making `dragColumnDivider` still take a one-element array of `dividerWidths` just for consistency / since the fact that there is only one divider isn’t fundamentally related to the fact that we’re working with columns here.